internal/generate/clusterserviceversion/clusterserviceversion_test.go

```diff
@@ -453,3 +506,19 @@ func upgradeCSV(csv *v1alpha1.ClusterServiceVersion, name, version string) *v1al
 
 	return upgraded
 }
+
+// removeSDKLabelsFromCSVString to remove the sdk labels from test CSV structs. The test
+// cases do not generate a PROJECTFILE or an entire operator to get the version or layout
+// of SDK. Hence the values of those will appear "unknown".
+func removeSDKLabelsFromCSVString(csv string) string {
```

Comment:
You can call this directly from `readFileHelper` so you don't have to add it to every test case.